tutorials/nservicebus-step-by-step/3-multiple-endpoints/Snippets/Core_8/Lesson3.cs

```diff
@@ -12,7 +12,7 @@ public async Task SendTypes(IEndpointInstance endpointInstance, IMessageHandlerC
 
             #region SendLocal
             // From endpoint startup code
-            await endpointInstance.SendLocal(command)
+            await endpointInstance.SendLocal(command, context.CancellationToken)
```

Comment:
These are included together for ease of writing the docs, but the token from the message handler context shouldn't be passed to the endpointInstance methods. Revert this and suppress the warning using an attribute on the SendTypes method. Set the justification in the attribute to be "Parameters only both included for ease of documentation"